site/content/docs/5.2/forms/input-group.md

```diff
@@ -64,6 +71,12 @@ Add the relative form sizing classes to the `.input-group` itself and contents w
 
 **Sizing on the individual input group elements isn't supported.**
 
+{{< ods-incompatibility-alert >}}
+This form variant, with **labels** that are **not above input fields**, should not be used because it does not respect the Orange Design System specifications.
```

Comment:
Maybe talk about the horizontal layout rather than "labels not above input fields".

Reply:
Same comment as above: We used the "with labels that are not above input fields" in severals other callout messages. I can replace everywhere by "with an horizontal layout", but even if less weird/heavy in the phrasing, to me it remains less clear/direct in the meaning.

Proposition with "horizontal layout":
> This form variant, with an **horizontal layout** (i.e. labels not above the input fields), should not be used because it does no respect the Orange Design System specifications.
>
>Please refer to the [Forms](https://system.design.orange.com/0c1af118d/p/88ab5b-forms/b/599459) guidelines and to the [Pages](https://system.design.orange.com/0c1af118d/p/20500e-form/b/16bb53) form examples on the Orange Design System website.

Reply:
Any updates ?

Reply:
Update done for the design callout message and OK.